lalrpop/build.rs

```diff
@@ -11,6 +11,17 @@ fn main() {
     }
 }
 
+fn find_lalrpop_binary(prefix: &str) -> Option<PathBuf> {
+    let lalrpop_path = Path::new(prefix)
+        .join("target/debug/lalrpop")
```

Comment:
Should this use the profile environment variable rather than hard coding debug?

Reply:
👍  Fixed.